src/components/MemoryBar.tsx

```diff
@@ -0,0 +1,32 @@
+import { ProgressBar } from '@/components/ProgressBar';
+import { formatInt } from '@/helpers/numbers';
+
+export function MemoryBar({
+  memoryUsed,
+  memoryAllocated,
+}: {
+  memoryUsed?: number | null | undefined;
+  memoryAllocated?: number | null | undefined;
+}) {
+  if (!memoryAllocated) {
+    return null;
+  }
+  const memoryUsedNum = memoryUsed || 0;
+  const mbRemaining = memoryAllocated - memoryUsedNum;
+  return (
+    <div className="margin-top-3" data-testid="memory-bar">
+      <p className="font-sans-3xs text-uppercase text-bold">Memory:</p>
+
+      <ProgressBar total={memoryAllocated} fill={memoryUsedNum} />
+
+      <div className="margin-top-1 display-flex flex-justify font-sans-3xs">
```

Comment:
When there’s a little too much text in these columns, it’d be nice to preserve some space between them:

<img width="328" alt="image" src="https://github.com/user-attachments/assets/aef5cb56-2274-48a6-9acc-f656080a57b8">

Is there a utility class that’d let us introduce a [`gap`](https://developer.mozilla.org/en-US/docs/Web/CSS/gap) here?

Reply:
I can put a 'lil margin so that the figures stack to 2 lines more quickly. How does this look?

![Screenshot 2024-10-09 at 2 25 29 PM](https://github.com/user-attachments/assets/b35019e9-bc88-4112-a628-ccfd59776cf3)

Reply:
i love that lil’ margin, it is so perfect and smol, _ship it_

Sudden and unwelcome scope creep: could we also add it between the columns above?

<img width="227" alt="image" src="https://github.com/user-attachments/assets/9261ee49-110e-4bc6-9a36-cda60a86088d">

Reply:
done!